site/en/docs/webstore/review-process/index.md

```diff
@@ -0,0 +1,362 @@
+such as malware distribution, deceptive behavior designed to evade review, repeated severe
+violations indicative of malicious intent, and other egregious policy violations, more drastic
+measures are necessary.
+
+In order to limit the potential for these developers to further harm users, the Chrome Web Store
+team intentionally does not provide details regarding these violations. Additionally, in more severe
+cases the developer's Chrome Web Store account will be permanently suspended.
+
+Developer communication
+
+: Unlike other enforcement actions, notification **is not** sent to the publisher's email address
+  when the extension(s) are taken down. In the event that the developer's Chrome Web Store account
+  is suspended, the developer will receive notification of that enforcement action.
+
+Chrome Web Store listing
+
+: Just as with a [takedown][header-takedown], the offending item is removed from the Chrome Web
+  Store.
+
+Chrome UI
+
+: The violating extension is disabled on all end-user devices. Unlike standard takedowns, these
+  extensions cannot be re-enabled. Chrome notifies the user that the extension has been disabled
+  because it was found to contain malware. Users may choose to remove the extension or dismiss the
+  dialog.
+
+## Developer communication {: #comms }
+
+The Chrome Web Store review process has two primary ways of communicating with developers: automated
+emails sent to the extension publisher's email address and support tickets.
+
+Support tickets must be opened using the [One Stop Support][header-support] form, but once a
+ticket is opened all communication takes place over email.
+
+### Automated emails {: #emails }
+
+In all but the most extreme policy violations, the Chrome Web Store will send developers automated
+emails informing them about the violation observed and the enforcement action taken. These emails
+state what policy or policies were violated, link to troubleshooting documentation related to the
+violation, and provide the developer with guidance on how to appeal the verdict.
+
+### One Stop Support {: #support }
+
+The [One Stop Support contact form][page-oss-form] provides Chrome Web Store publishers with a
+single contact point to request assistance with a variety of issues.
+
+{% Details %}
+{% DetailsSummary %}
+
+Appealing a review verdict
+
+{% endDetailsSummary %}
+
+1. Open the [One Stop Support contact form][page-oss-form].
+2. Select "My item (extensions, app, or theme)"
+3. Select "My item was warned / removed / rejected"
+4. Select why you are appealing, and the reference color and element.
+5. Review the violation troubleshooting guidance.
+6. Provide additional details as requested by the form.
+
+{% endDetails %}
+
+{% Details %}
+{% DetailsSummary %}
+
+Appealing an account suspension
+
+{% endDetailsSummary %}
+
+1. "My developer account"
+2. "My account was suspended".
+3. Provide additional details as requested by the form.
```

Comment:
Are these instructions or this this a list. If it's instructions, it needs verbs. If it's a list it needs to be bullets, not numbers. If the latter, it needs an intro explaining what this is a list of.

Reply:
This was meant as instructions. I don't recall writing these, so I *think* they may have been added by Andy.

In any case, I'm updating this list to use the same structure as the previous one and adding an intro sentence that sets context.